dev/import-beats/icons.go

```diff
@@ -99,11 +99,11 @@ func retrieveIconPathFromTutorials(kibanaDir string) (map[string]string, error)
 		// Extracting module name from tutorials path
 		// e.g. ./src/plugins/home/server/tutorials//php_fpm_metrics/index.ts -> php_fpm
 		moduleName := tutorialFilePath[len(tutorialsPath)+1:]
-		moduleName = moduleName[:strings.Index(moduleName, "/")]
+		moduleName = moduleName[:strings.Index(moduleName, string(os.PathSeparator))]
 		moduleName = moduleName[:strings.LastIndex(moduleName, "_")]
 
-		if val[0] == '/' {
-			iconFileName := val[strings.LastIndex(val, "/")+1:]
+		if val[0] == os.PathSeparator {
+			iconFileName := val[strings.LastIndex(val, string(os.PathSeparator))+1:]
```

Comment:
Is this checking if this is an absolute path, and then taking the base name?
```suggestion
		if filepath.IsAbs(val) {
			iconFileName := filepath.Base(val)
```

Btw, does it make sense a path in Windows that starts with `\`?